index.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="en-us">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <title>Horiseon Marketing Agency Landing Page</title>
+  </head>
+
+  <body>
+    <!-- Semantic HTML:The logo and navigation links (nav bar) is nested in a header tag -->
+
+    <header class="header">
+      <h1>Hori<span class="seo">seo</span>n</h1>
+
+      <nav>
+        <ul>
+          <li>
+            <a href="#search-engine-optimization">Search Engine Optimization</a>
+          </li>
+          <li>
+            <a href="#online-reputation-management"
+              >Online Reputation Management</a
+            >
+          </li>
+          <li>
+            <a href="#social-media-marketing">Social Media Marketing</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <!--Semantic HTML: The large background image is nested in a section tag  -->
+
+    <div class="hero"></div>
+
+    <!--Semantic HTML: The 3 independent self contained content is nested in an article tag which is then grouped together and nested in a main tag showing the main content of the web-page -->
+
+    <!--Semantic HTML: The img and figcaption elements are nested in a figure tag and for accessibility reasons an alt attribute is added -->
+
+    <main class="content">
+      <!--added id element in order to link the search engine bar on the nav bar -->
+
+      <article
```

Comment:
A section tag could be used here instead of an article tag. Article tags are usually used when the content is independent of the content on the page. A section tag would be more appropriate as the content in each section is related. Read more [here](https://www.geeksforgeeks.org/difference-between-article-tag-and-section-tag/). Amend this for all other articles please.